scripts/ubuntu/1-install-dependendcies.sh

```diff
@@ -64,4 +96,9 @@ done
 IFS=$SAVEIFS 
 
 # Install Peotry, needed for Test Harness CLI
+echo
+echo "####################################"
+echo "Install Peotry, needed for Test Harness CLI"
```

Comment:
```suggestion
echo "Install Poetry, needed for Test Harness CLI"
```

Reply:
This has been fixed in a separate commit